clevertap-core/src/main/java/com/clevertap/android/sdk/pushnotification/PushProviders.java

```diff
@@ -937,7 +937,7 @@ private void createAlarmScheduler(Context context) {
             intent.setPackage(context.getPackageName());
 
             int flagsAlarmPendingIntent = PendingIntent.FLAG_UPDATE_CURRENT;
-            if (VERSION.SDK_INT >= VERSION_CODES.M) {
+            if (VERSION.SDK_INT >= VERSION_CODES.S) {
```

Comment:
@CTLalit if we make this change now then we will need to re-release the core Android SDK as well. Can we exclude this change for now?